### PR TITLE
Katello 4.12 Release Notes

### PR DESCRIPTION
**Pull request by qcjames53**
* [X] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* We do not accept PRs for Foreman older than 3.3.

Added Katello 4.12 release notes and contributors.

### CODE REVIEW

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (jeremylenz) [inline guides/doc-Release_Notes/topics/katello.adoc:7]:
```suggestion
=== Entitlement Mode Removed in Favor of Simple Content Access
```

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (jeremylenz) [inline guides/doc-Release_Notes/topics/katello.adoc:32]:
```suggestion
Simple Content Access is now the only supported content access mode. Existing organizations will be automatically migrated on Katello upgrade (see more info above). Please note that the 'Subscription - Entitlements' report template does not work with SCA organizations, and will never have any rows. It will be removed in a future release or release candidate. Instead, use the 'Host - Installed Products' report. Additionally, please note that SCA-related API and Hammer endpoints and parameters have not yet been removed, so you will experience errors if you try to use them. They will be removed in a future release or release candidate.
```

**Review by jeremylenz — COMMENTED** [review]
SCA-Only content is looking great, just a couple nits :)

**Comment by qcjames53** [comment]
@jeremylenz Rolled in your suggested changes, thanks. Also fixed a merge conflict.

File: guides/doc-Release_Notes/master.adoc
Comment (qcjames53) [reply guides/doc-Release_Notes/master.adoc:17]: I manually merged this PR with upstream when 3.10 was added so the order was reversed. Will fix.

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (qcjames53) [reply guides/doc-Release_Notes/topics/katello.adoc:32]: I have separated the paragraphs into individual sentences, should be fixed. Thanks for the heads up.

**Comment by qcjames53** [comment]
@ekohl Thank you for all the rec's for the documentation on Katello 4.12. I have rebased the PR to include your suggestions. I split the contributors list into lines containing just people's names but if there should be another separator please let me know. That file has been blank for a release or two at this point.

**Comment by qcjames53** [comment]
@ekohl Did you want to take another look at this or are we ok to merge the changes in?

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (bangelic) [inline guides/doc-Release_Notes/topics/katello.adoc:8]:
```suggestion
Simple Content Access (SCA) is now the only content access mode supported.
```

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (bangelic) [inline guides/doc-Release_Notes/topics/katello.adoc:25]:
```suggestion
Additionally, the host details page will now display more details for debian packages.
```

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (bangelic) [inline guides/doc-Release_Notes/topics/katello.adoc:30]:
```suggestion
Simple Content Access (SCA) is now the only supported content access mode.
```

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (bangelic) [inline guides/doc-Release_Notes/topics/katello.adoc:32]:
```suggestion
Please note that the 'Subscription - Entitlements' report template does not work with SCA organizations and will never have any rows.
```

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (bangelic) [inline guides/doc-Release_Notes/topics/katello.adoc:35]:
```suggestion
Additionally, please note that SCA-related API and Hammer endpoints and parameters have not yet been removed so you will experience errors if you try to use them.
```

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (bangelic) [inline guides/doc-Release_Notes/topics/katello.adoc:47]:
```suggestion
Deprecated certain SCA-related API and Hammer endpoints and parameters.
```

**Review by bangelic — CHANGES_REQUESTED** [review]

**Comment by qcjames53** [comment]
@ekohl Did you want to take another look at the Katello 4.12 release documentation before we merge it in? We're releasing 4.12 today (hopefully) so we will likely need to merge this afternoon.

File: guides/doc-Release_Notes/topics/katello-contributors.adoc
Comment (ekohl) [inline guides/doc-Release_Notes/topics/katello-contributors.adoc:1]: I think you've listed every contributor ever, instead of just those who contributed in 4.12.

**Review by ekohl — CHANGES_REQUESTED** [review]

File: guides/doc-Release_Notes/topics/katello-contributors.adoc
Comment (qcjames53) [reply guides/doc-Release_Notes/topics/katello-contributors.adoc:1]: Thanks for taking a look at this. That is correct; I was unaware that this is only 4.12 contributors. I'll update this soon.

Comment (ekohl) [reply guides/doc-Release_Notes/topics/katello-contributors.adoc:1]: I hope you are aware of https://github.com/theforeman/theforeman.org/blob/gh-pages/scripts/committers.rb. I'd also like to discuss merging the contributor lists together into one list, but that's a separate discussion.

Comment (qcjames53) [reply guides/doc-Release_Notes/topics/katello-contributors.adoc:1]: Thanks Ewoud. The contributors list should now be correct for 4.12 alone. Ok for a re-review.

That original big list of all contributors was generated from commands but it needed a decent bit of manual sleuthing since a few of the commits has incorrect names, nicknames, or emails as the full name field.

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (bangelic) [inline guides/doc-Release_Notes/topics/katello.adoc:46]:
```suggestion
```

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (bangelic) [inline guides/doc-Release_Notes/topics/katello.adoc:21]: Was it spelled like this in the UI? Not spelled Add-on or Add on?

Comment (jeremylenz) [reply guides/doc-Release_Notes/topics/katello.adoc:21]: Unfortunately yes :confounded: 

image

Comment (qcjames53) [reply guides/doc-Release_Notes/topics/katello.adoc:21]: Addons status sounds icky to me. Just like fathers in law. I think it's ok as is?

File: guides/doc-Release_Notes/topics/katello-contributors.adoc
Comment (ekohl) [reply guides/doc-Release_Notes/topics/katello-contributors.adoc:1]: In https://github.com/theforeman/theforeman.org/blob/gh-pages/.mailmap I did make that mapping, but only for "recent" contributors. If you have more manual fixes you need to apply, that's the place to fix it.

**Review by ekohl — APPROVED** [review]

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (jeremylenz) [reply guides/doc-Release_Notes/topics/katello.adoc:21]: https://www.theonion.com/william-safire-orders-two-whoppers-junior-1819565735

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (ianballou) [reply guides/doc-Release_Notes/topics/katello.adoc:46]: I think we need the extra space there to logically separate the subs entitlement report template deprecation from the the next one:

image

**Review by ianballou — APPROVED** [review]
Looking good to me!

**Comment by ianballou** [comment]
Merging so we can be on time with GA since we have two acks -- if further improvements are requested we can include them in a separate PR.

**Pull request merged**
